src/docs/asciidoc/http-server.adoc

```diff
@@ -685,3 +685,108 @@ HTTP/2 200
+| reactor.netty.http.server.errors | Counter | Number of the errors that are occurred
+| reactor.netty.http.server.tls.handshake.time | Timer | Time that is spent for TLS handshake
+| reactor.netty.http.server.data.received.time | Timer | Time that is spent in consuming incoming data
+| reactor.netty.http.server.data.sent.time | Timer | Time that is spent in sending outgoing data
+| reactor.netty.http.server.response.time | Timer | Total time for the request/response
+|=======
+
+These additional metrics also will be available:
+
+include::alloc-metrics.adoc[]
+
+The following example enables that integration:
+
+====
+[source,java]
+----
+import io.micrometer.core.instrument.Metrics;
+import io.micrometer.core.instrument.config.MeterFilter;
+import reactor.core.publisher.Mono;
+import reactor.netty.DisposableServer;
+import reactor.netty.http.server.HttpServer;
+
+public class Application {
+
+    public static void main(String[] args) {
+        Metrics.globalRegistry <1>
+               .config()
+               .meterFilter(MeterFilter.maximumAllowableTags("reactor.netty.http.server", "URI", 100, MeterFilter.deny()));
+
+        DisposableServer server =
+                HttpServer.create()
+                          .metrics(true, s -> {
+                              if (s.startsWith("/stream/")) { <2>
+                                  return "/stream/{n}";
+                              }
+                              else if (s.startsWith("/bytes/")) {
+                                  return "/bytes/{n}";
+                              }
+                              return s;
+                          }) <3>
+                          .route(r ->
+                                  r.get("/stream/{n}",
+                                        (req, res) -> res.sendString(Mono.just(req.param("n"))))
+                                   .get("/bytes/{n}",
+                                        (req, res) -> res.sendString(Mono.just(req.param("n")))))
+                          .bindNow();
+
+        server.onDispose()
+              .block();
+    }
+}
+----
+<1> Applies upper limit for the meters with `URI` tag
+<2> Templated URIs will be used as an URI tag value when possible
+<3> Enables the built-in integration with Micrometer
+====
+
+NOTE: In order to avoid an overhead of the enabled metrics it is important to convert the real URIs to templated URIs when possible.
+Without a conversion to a template-like form, each distinct URI leads to the creation of a distinct tag, which takes a lot of memory for the metrics.
+
+NOTE: Always apply upper limit for the meters with URI tag. Configuring an upper limit on the number of meters can help in cases when the real URIs cannot be templated.
+More information here https://micrometer.io/docs/concepts#_denyaccept_meters[maximumAllowableTags]
+
+When HTTP server metrics are needed, but with an integration with a system different than `Micrometer` or you want
```

Comment:
Change it to the following: When HTTP client metrics are needed for an integration with a system other than Micrometer or you want
to provide your own integration with Micrometer, you can provide your own metrics recorder, as follows:

(Same comment as earlier.)